cmd/scheduler/app/options/options.go

```diff
@@ -102,6 +108,7 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&o.EnableEmptyWorkloadPropagation, "enable-empty-workload-propagation", false, "Enable workload with replicas 0 to be propagated to member clusters.")
 	fs.StringSliceVar(&o.Plugins, "plugins", []string{"*"},
 		fmt.Sprintf("A list of plugins to enable. '*' enables all build-in and customized plugins, 'foo' enables the plugin named 'foo', '*,-foo' disables the plugin named 'foo'.\nAll build-in plugins: %s.", strings.Join(frameworkplugins.NewInTreeRegistry().FactoryNames(), ",")))
+	fs.StringVar(&o.SchedulerName, "scheduler-name", scheduler.DefaultScheduler, "SchedulerName represents the name of the scheduler. default is 'default-scheduler'.")
```

Comment:
I wonder how `kube-scheduler` know its name. I didn't find the relevant name in the document:
https://kubernetes.io/docs/reference/command-line-tools-reference/kube-scheduler/

Reply:
One `kube-scheduler` process can load multiple scheduling profiles via `KubeSchedulerConfiguration`, e.g.:
```yaml
apiVersion: kubescheduler.config.k8s.io/v1beta1
kind: KubeSchedulerConfiguration
profiles:
- schedulerName: default-scheduler
  pluginConfig: {...}
- schedulerName: custom-scheduler
  pluginConfig: {...}
```